modules/database/src/adapters/DatabaseAdapter.ts

```diff
@@ -211,6 +212,39 @@ export abstract class DatabaseAdapter<T extends Schema> {
     return true;
   }
 
+  async compareAndStoreMigratedSchema(schema: ConduitSchema) {
+    if (['_DeclaredSchema', 'MigratedSchemas'].includes(schema.name)) {
+      return;
+    }
+    const storedSchema = await this.models['_DeclaredSchema'].findOne({
+      name: schema.name,
+    });
+    if (isNil(storedSchema)) {
+      return;
+    }
+    if (isEqual(schema.fields, storedSchema.fields)) {
+      return;
+    }
+    const lastMigratedSchemas = await this.models['MigratedSchemas'].findMany(
+      { name: storedSchema.name },
+      undefined,
+      1,
+      undefined,
+      { version: -1 },
+    );
+    const lastVersion =
```

Comment:
what we want to store is the schema itself, so fields, modeloptions etc. Here you are fetching all the data of the schema (ex. for User schema you would fetch all the users) and saving it.